deploy/charts/cert-manager/templates/webhook-deployment.yaml

```diff
@@ -80,6 +80,9 @@ spec:
           {{- with .Values.webhook.extraArgs }}
           {{- toYaml . | nindent 10 }}
           {{- end }}
+          { { - if .Values.featureGates } }
+          - --feature-gates={{ .Values.featureGates }}
+          { { - end } }
```

Comment:
The only problem is that each of the three components: controller, webhook, cainjector have their own set of features gates.
For example, the controller gate `ValidateCAA` would be unrecognized by the webhook and cause an error I think.

 * https://github.com/cert-manager/cert-manager/blob/master/internal/controller/feature/features.go
 * https://github.com/cert-manager/cert-manager/blob/master/internal/webhook/feature/features.go

In our E2E tests we set the webhook and cainjector feature flags using the ExtraArgs Helm value.
Would that work for you too?

https://github.com/cert-manager/cert-manager/blob/25387770ebf70bf37fe43935fbdd52ce39bc2970/devel/addon/certmanager/install.sh#L95-L96